docs/api_requests/logger-and-metrics.md

```diff
@@ -1,15 +1,20 @@
-# Logger API Request
+# Logger and Metrics API Requests
 
 The Logger can be configured by sending a `PUT` API Request to the `/logger`
 path.
-A minimal logger configuration example is the following:
+The Metrics system can be configured by sending a `PUT` API Request to the
+`/metrics` path.
+You can configure none, one or both of these systems as they are not
```

Comment:
I was thinking to mention this because I am talking about both of these systems at the same time in this file and also because they were interdependent before.

Reply:
> I was thinking to mention this because I am talking about both of these systems at the same time in this file and also because they were interdependent before.

It is something more like a nit. It is fine if this will present the usage more clear to users.